app/authentication/authenticator.py

```diff
@@ -103,7 +103,10 @@ def load_user(extend_session: bool = True) -> Optional[User]:
         user_ik = cookie_session.get(USER_IK)
         user = User(user_id, user_ik)
 
-        if session_store.session_data.tx_id:
+        if (
+            isinstance(session_store.session_data, SessionData)
```

Comment:
Because `session_data` had a type `Union[dict, SessionData]` in `globals.py`, I've modified it in globals (removed dict as a potential type), so it only can be optional SessionData and we check if it exists in this snippet above now instead of `isinstance()`. Thanks!